components/event-bus/internal/knative/subscription/controller/knativesubscription/reconcile.go

```diff
@@ -3,87 +3,80 @@ package knativesubscription
 import (
 	"context"
 
-	evapisv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
-	subApis "github.com/kyma-project/kyma/components/event-bus/api/push/eventing.kyma-project.io/v1alpha1"
-	"github.com/kyma-project/kyma/components/event-bus/internal/knative/util"
+	pkgerrors "github.com/pkg/errors"
+	"go.uber.org/zap"
+
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/client-go/tools/record"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/tools/cache"
+
+	messagingapisv1alpha1 "knative.dev/eventing/pkg/apis/messaging/v1alpha1"
+	messagingv1alpha1 "knative.dev/eventing/pkg/apis/messaging/v1alpha1"
+	subscriptionlistersv1alpha1 "knative.dev/eventing/pkg/client/listers/messaging/v1alpha1"
+	"knative.dev/eventing/pkg/logging"
+	"knative.dev/eventing/pkg/reconciler"
 	"knative.dev/pkg/apis"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"knative.dev/pkg/controller"
+
+	subApis "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+	kymaeventingv1alpha1 "github.com/kyma-project/kyma/components/event-bus/client/generated/clientset/internalclientset/typed/eventing/v1alpha1"
+	"github.com/kyma-project/kyma/components/event-bus/internal/knative/util"
 )
 
 const (
-	finalizerName = "subscription.finalizers.kyma-project.io"
+	finalizerName                 = "subscription.finalizers.kyma-project.io"
+	knativesubscriptionreconciled = "KnativeSubscriptionReconciled"
```

Comment:
What about `KnativeSubscriptionReconcileFailed` and `SubscriptionReconcileFailed` (in the same file)?

Reply:
@antoineco Should they both be a single constant, `"KnativeSubscriptionReconcileFailed"`?

Reply:
I mean, if we use a constant for one event description, we should use it for all, otherwise it's inconsistent.